packages/api/src/routers/svc/links.ts

```diff
@@ -52,5 +80,38 @@ export function linkServiceRouter() {
     }
   })
 
+  router.post('/prune', async (req, res) => {
+    logger.info('prune expired items in folder')
+
+    const { message: msgStr, expired } = readPushSubscription(req)
```

Comment:
This one will validate the token, right?

Reply:
Yeah, validate the token from the pubsub payload